tests/kafka/kafka.test.ts

```diff
@@ -142,8 +152,22 @@ class DBOSTestClass {
   static async testWorkflow(_ctxt: WorkflowContext, topic: string, _partition: number, message: KafkaMessage) {
     if (topic == wfTopic && message.value?.toString() === wfMessage) {
       wfCounter = wfCounter + 1;
-      DBOSTestClass.wfResolve()
+      DBOSTestClass.wfResolve();
     }
     await DBOSTestClass.wfPromise;
   }
+
+  @KafkaConsume(patternTopic)
+  @Workflow()
+  static async testConsumeTopicsByPattern(_ctxt: WorkflowContext, topic: string, _partition: number, message: KafkaMessage) {
+    const isWfMessage = topic == wfTopic && message.value?.toString() === wfMessage;
+    const isTxnMessage = txnTopic && message.value?.toString() === txnMessage;
+    if ( isWfMessage || isTxnMessage ) {
+      patternTopicCounter = patternTopicCounter + 1;
+      if (patternTopicCounter === 2) {
+        DBOSTestClass.patternTopicResolve();
+      }
+    }
+    return DBOSTestClass.patternTopicPromise;
```

Comment:
Don't return promises from operations, just await them like in the other tests

Reply:
It is not a big deal here cause it is just a test but why await? the caller awaits if he cares to. I ve learned not to await unless needed (can very easily create synchronous code where you don't expect it ). 
As you wish though.

Reply:
I also have learned to use explicit returns. (burned sometimes for not doing it in nestjs).